docs/src/utils/stage.ts

```diff
@@ -1,7 +0,0 @@
-const PROD_STAGE_SITE_URL = 'https://ui.docs.amplify.aws';
-const DEV_STAGE_SITE_URL = 'dev.ui.docs.amplify.aws';
```

Comment:
Are we sure this isn't used anywhere else?

Reply:
+1

Reply:
yup, did a search through the repo. And we will see the docs build test fail if it is used somewhere.